server/services/PopulatedData/populateData.js

```diff
@@ -24,11 +24,8 @@ const getTests = require('../../models/services/TestsService');
  * and no database queries will be run by this function.
  * @returns
  */
-const populateData = async (
-    locationOfData,
-    { transaction, preloaded } = {}
-) => {
-    const context = { transaction };
+const populateData = async (locationOfData, { context, preloaded } = {}) => {
+    const { transaction } = context;
```

Comment:
Curious about the reasoning behind changing the API to accept the `context` instead of the transaction?

Reply:
Now that context is a required field for resolvers, this is needed by populateData so that it can call out to any resolvers. And in fact it does call the `testPlanVersionTestPlanResolver` below.